src/compression/compression.py

```diff
@@ -2,21 +2,24 @@
 import logging, os
 logger = logging.getLogger(__name__)
 
+
 def gunzip_file(**kwargs):
     logger.info('{}: Extracting file with gunzip: {}'.format(kwargs['worker'], kwargs['filename']))
     os.system("gunzip -f {}/{}".format(kwargs['savepath'], kwargs['filename']))
 
+
 def pass_func(**kwargs):
     logger.info('{}: Skipping decompression for {}.'.format(kwargs['worker'], kwargs['filename']))
     pass
 
+
 # Main function below.
 def decompress(worker, filename, savepath):
     # After writing functions for decompression, please add them to this dispatch table by suffix.
     dispatch_dict = {
-        'gz' : gunzip_file,
-        'json' : pass_func
+        'gz': gunzip_file,
+        'json': pass_func
     }
```

Comment:
This is where I'd like to keep all our file suffixes and decompression logic, if possible.

Reply:
Each value in the dictionary is the name of a function to use to deal with the file suffix.

Reply:
for this, I just removed the space to allow my editor to settle down. :)